custom_components/sleepme_thermostat/config_flow.py

```
@@ -1,132 +1,84 @@
-            # Instantiate SleepMeClient to get the list of devices
-            client = SleepMeClient(API_URL, self.api_token)
-
+            self.api_token = user_input[CONF_API_TOKEN]
             try:
-                # Get the list of claimed devices
-                self.claimed_devices = await client.get_claimed_devices()
-                _LOGGER.debug(f"Claimed devices: {self.claimed_devices}")
-
-                if not self.claimed_devices:
+                client = SleepMeClient(DEFAULT_API_URL, self.api_token, None)
+                all_devices = await client.get_all_devices()
```

Pylint (3.3.7)

[error] Instance of 'SleepMeClient' has no 'get_all_devices' member

(E1101)